packages/react-native/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
```

Comment:
Should this become a react-native release?

Reply:
Nope, i only updated the changelog here to not forget.